### PR TITLE
Add profile edit flow

### PR DESCRIPTION
**Pull request by maxious**

### CODE REVIEW

**Comment by nathf** [comment]
A lot of duplication here, whats the difference between this and the original?

**Comment by maxious** [comment]
Hmm, I've added a different implementation using filterSteps which still differentiates the entrypoint if we need to diverge more later.

File: src/shared/SellerProfile/Body.js [hunk 1]
```diff
@@ -120,7 +120,7 @@ const Body = (props) => {
       </Row>
 
       <Row title="Documents" show={!isEmpty(documents)}>
-          {Object.keys(documents).map((key, val) => {
+          {!isEmpty(documents) && Object.keys(documents).map((key, val) => {
```
Comment (nathf) [inline src/shared/SellerProfile/Body.js]: Is this required? Shouldn't the show prop on Row cater for this case?

Comment (maxious) [inline src/shared/SellerProfile/Body.js]: Yes, without it empty documents was still crashing render. I'm guessing it's still evaluated but passed to Row which then decides if it's not rendered?

Comment (nathf) [inline src/shared/SellerProfile/Body.js]: Weird, I'll take a look later

**Pull request merged**